src/main/java/ch/njol/skript/events/EvtStatisticChange.java

```diff
@@ -0,0 +1,134 @@
+				)
+				.examples(
+						"on player statistic increase:",
+							"\tsend \"Statistic increased: %event-string% from %past event-number% to %future event-number% (diff: %event-number%)\" to player",
+							"\tif event-entitytype is set:",
+								"\t\tsend \"Of entity: %event-entitytype%\" to player",
+							"\telse if event-item is set:",
+								"\t\tsend \"Of item: %event-item%\" to player",
+						"",
+						"on player statistic increase of \"CHEST_OPENED\":",
+							"\tsend \"You just opened a chest!\" to player"
+				)
+				.since("INSERT VERSION");
+
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, String.class, new Getter<String, PlayerStatisticIncrementEvent>() {
+			@Nullable
+			@Override
+			public String get(PlayerStatisticIncrementEvent event) {
+				return event.getStatistic().toString();
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, Number.class, new Getter<Number, PlayerStatisticIncrementEvent>() {
+			@Override
+			public Number get(PlayerStatisticIncrementEvent event) {
+				return event.getPreviousValue();
+			}
+		}, EventValues.TIME_PAST);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, Number.class, new Getter<Number, PlayerStatisticIncrementEvent>() {
+			@Override
+			public Number get(PlayerStatisticIncrementEvent event) {
+				return event.getNewValue() - event.getPreviousValue();
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, Number.class, new Getter<Number, PlayerStatisticIncrementEvent>() {
+			@Override
+			public Number get(PlayerStatisticIncrementEvent event) {
+				return event.getNewValue();
+			}
+		}, EventValues.TIME_FUTURE);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, EntityData.class, new Getter<EntityData, PlayerStatisticIncrementEvent>() {
+			@Nullable
+			@Override
+			public EntityData<?> get(PlayerStatisticIncrementEvent event) {
+				Class<? extends Entity> c = event.getEntityType() != null ? event.getEntityType().getEntityClass() : null;
+				return c == null ? null : EntityData.fromClass(c);
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(PlayerStatisticIncrementEvent.class, ItemStack.class, new Getter<ItemStack, PlayerStatisticIncrementEvent>() {
+			@Nullable
+			@Override
+			public ItemStack get(PlayerStatisticIncrementEvent event) {
+				return event.getMaterial() == null ? null : new ItemStack(event.getMaterial());
+			}
+		}, EventValues.TIME_NOW);
+	}
+
+	private final List<Statistic> statistics = new ArrayList<>(0);
+
+	@Override
+	public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult) {
+		if (args[0] == null)
+			return true;
+		statistics.addAll(Arrays.asList((Statistic[]) args[0].getArray()));
```

Comment:
I don't think it matters here (or maybe at all), but in general should we be using getArray or getAll for getting literals for events?

Reply:
I updated the whole checking technique, check it now

Reply:
Looks great! I doubt there's any behavioral difference but it certainly looks a lot nicer!

Reply:
Works perfectly :ok_hand: 
I only noticed we can't use `on statistics change of "JUMP" and "DROP_COUNT":` anymore, we have to use `on statistics change of "JUMP" or "DROP_COUNT":` to get it to work

Reply:
Actually, it probably behaves differently with AND lists, but i don't think it's ever possible for the event to have increased more than one statistic, so I think it just wouldn't trigger if you gave it an AND list?